packages/components/src/form/RadioGroup/RadioGroup.js

```diff
@@ -29,6 +29,7 @@ export const RADIOGROUP_DEFAULT_PROPS = {
   value: '',
   imageHeight: 100,
   fullWidth: false,
+  minWidth: false,
```

Comment:
If minWidth is a number as defined in propTypes it should be treated as so

```suggestion
  minWidth: 0,
```